Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 pipeline {
   agent {
         docker {
-          image 'nemo_containers:23.01_apex_c3d575f2478cd379b3c2d81f41edde39791b5d92'
+          image 'nvcr.io/nvidia/pytorch:23.02-py3'
```

Comment:
is it on purpose to switch to a Pytorch container from the nemo container?

Reply:
ah I see. Thank your for explanation.

Reply:
Yes I'm sorry for this confusion. But we're only taking the PyTorch base container and installing different version of apex. It's not actually a nemo container, I will change the name